runtime/src/eval/macros.rs

```diff
@@ -69,3 +69,21 @@ macro_rules! pop_usize {
 		)*
 	);
 }
+
+macro_rules! as_usize_or_fail {
+	( $v:expr ) => {{
+		if $v > U256::from(usize::MAX) {
```

Comment:
This could be more efficient if you define a constant

```
const USIZE_MAX: U256 = U256([usize::MAX as u64, 0, 0, 0]);
```

Reply:
Refactored here: [b1dcd90](https://github.com/aurora-is-near/sputnikvm/pull/40/commits/b1dcd909780b32a7498373b97d61e046d50b9687)